### ISSUE TITLE
Scoped packages should be displayed using `/` separator, regardless of OS

### ISSUE THREAD

**Issue by nelson6e65** [ranyitz/qnm#98]
On Windows, `qnm` displays scoped packages using `\` separator instead of the name of the package on npm register. It should use `@scope/name` instead of `@scope\name` regardless of directory separator used by the operating system.

**Example:**

```ps
$ qnm doctor
size    count  module
------  -----  ----------
107.7mb   2 😠 typescript
59.42mb   1 ✓  @firebase\firestore
49.97mb   1 ✓  cordova-plugin-ionic
41.68mb  55 😖 source-map
20.78mb   1 ✓  @angular\core
20.31mb   2 😠 @schematics\angular
18.74mb   1 ✓  @angular\compiler
18.57mb  37 😖 postcss
18.16mb   1 ✓  firebase
16.83mb   1 ✓  prettier
15.65mb   1 ✓  @ionic\core
15.47mb   1 ✓  @stencil\core
13.46mb   3 😠 rxjs
12.86mb   1 ✓  @firebase\database
12.44mb   1 ✓  @angular\common
12.16mb   1 ✓  @ionic\angular-toolkit
11.71mb  13 😡 ajv
9.36mb    1 ✓  @angular\compiler-cli
8.79mb    1 ✓  esbuild-wasm
8.54mb    1 ✓  npm
```

**Comment by ranyitz** [comment]
Hey @nelson6e65, thanks for opening this issue!

It seems that the [logic on this line](https://github.com/ranyitz/qnm/blob/71bec8951d356efcfea5458878dcd69b2f93271d/src/workspace/modules-map.ts#L144) is wrong, instead of using the `path` module, we should just join the directory name and the module name with `/` regardless of the environment (windows/linux/mac).

I believe this will solve both this issue and https://github.com/ranyitz/qnm/issues/97

@nelson6e65 would you be interested in contributing a PR that fixes this?

**Comment by nelson6e65** [comment]
Sure.

**Comment by nelson6e65** [comment]
Hello, @ranyitz I have a pre-pr 😅

I detected an issue when I cloned the repository to a directory with spaces. Can you check if my "solution" on #99 is Ok?

**Comment by nelson6e65** [comment]
I don't know if you want I complete all Windows compatibility or merge #99 as a pre-fix (some tests are able to run, but some are still failing on Windows). 😅

**Comment by ranyitz** [comment]
Fixed on #100 and released on `v2.6.0`
